src/main/java/interface_adapter/home/HomeState.java

```diff
@@ -0,0 +1,11 @@
+package interface_adapter.home;
+import java.util.ArrayList;
+public class HomeState {
+    private ArrayList<String> wantedPosts = new ArrayList<>();
```

Comment:
I think this would be better as an ArrayList<Item> instead of an ArrayLst<String> to account for an interactive Home page, as per our discussion today